policy_agent/policy_agent.py

```diff
@@ -0,0 +1,698 @@
+#!/usr/bin/python
+import time
+import json
+import Queue
+import logging
+from threading import Thread
+
+import requests
+
+import common
+from common.constants import *
+from common.util import configure_logger, _get_api_stream, _get_api_list
+import pycalico
+from pycalico.datastore_datatypes import Rules, Rule, Profile
+from pycalico.datastore_errors import ProfileNotInEndpoint, ProfileAlreadyInEndpoint
+from pycalico.datastore import DatastoreClient
+
+logger = logging.getLogger(__name__)
+util_logger = logging.getLogger(common.util.__name__)
+pycalico_logger = logging.getLogger(pycalico.__name__)
+
+_datastore_client = DatastoreClient()
+
+
+class PolicyAgent():
```

Comment:
Shouldn't this class (and the Resource class I spotted further down) be subclassing from `object`?